src/main/resources/templates/questionnaire.html

```diff
@@ -0,0 +1,161 @@
+{% extends "base.html" %}
+
+{% block title %}Mentor Matching{% endblock %}
+
+{% block styles %}
+  {{ super() }}
+  {% set styles = [
+      "static/styles/questionnaire.css"
+    ] %}
+  {% for style in styles %}
+    <link rel="stylesheet" type="text/css" href={{ style }}>
+  {% endfor %}
+{% endblock %}
+
+{% block main_content %}
+
+<div class = "form-container">
+  <form action="action_page.php">
+    <h3> Step 1: Basic Personal Information </h3>
+    <label for="fname">First Name</label>
+    <input type="text" id="fname" name="firstname" placeholder="Your name.." required>
+
+    <label for="lname">Last Name</label>
+    <input type="text" id="lname" name="lastname" placeholder="Your last name.." required>
+
+    <label for="dob">Date of Birth:</label>
+    <input type="date" name="dob" required>
+
+    <label for="country">Country</label>
+    <select id="country" name="country" required>
+      {% for country in countries %}
+      <option value="{{country}}">{{country}}</option>
+      {% endfor %}
+    </select>
+
+    <label for="state">State</label>
+    <select id="state" name="state" required>
+      {% for state in states %}
+    	<option value="{{state.abbreviation}}">{{state.name}}</option>
+      {% endfor %}
+    </select>
+
+
+
+    <label for="city">City</label>
+    <input type="text" id="city" name="city" required>
+    <h3> Step 2: More Personal Information </h3>
+    <p> This information is used to find a person that best fits you.
+       Feel free to fill in with whatever you're comfortable sharing: everything in this step is optional </p>
+    <div id="ethnicity">
+      <p> Check off whatever best applies to you </p>
+      <input type="checkbox" name="asian" value="asian">
+      <label for="asian"> Asian</label><br>
+      <input type="checkbox" name="black-african" value="black-african">
+      <label for="black-african"> Black/African</label><br>
+      <input type="checkbox" name="caucasian" value="caucasian">
```

Comment:
prefer to have these multiple choice options generated in the backend this way the list of valid options is always in sync with what the backend can accept.